app/connectors/HIP/BroughtForwardLossConnector.scala

```diff
@@ -27,12 +28,12 @@ import utils.Logging
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.ExecutionContext
 
-trait HipConnector {
+trait BroughtForwardLossConnector {
```

Comment:
These traits are unnecessary. They were only created to support the "Stub" classes, which we're in the process of replacing with Mockito mocks.

Please remove